bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -97,15 +97,28 @@ The *landline* thing requires the following configuration parameters:
 |------------------|-----------------|------------------------------------------------------------------------|----------|---------|
 | Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll for phone state. | No       | 2       |
 
-### Network devices: Host and WifiHost
+### Network devices: Host
 
-The *host* and *wifihost* things requires the following configuration parameters:
+The *host* things requires the following configuration parameters:
 
 | Parameter Label  | Parameter ID    | Description                                                            | Required | Default |
 |------------------|-----------------|------------------------------------------------------------------------|----------|---------|
-| MAC Address      | macAddress      | The MAC address of the network host .                                  | Yes      |         |
+| MAC Address      | macAddress      | The MAC address of the network host.                                   | Yes      |         |
 | Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll for phone state. | No       | 30      |
 
+### Network devices: WifiHost
+
+The *wifihost* things requires the following configuration parameters:
+
+| Parameter Label  | Parameter ID    | Description                                                            | Required | Default |
+|------------------|-----------------|------------------------------------------------------------------------|----------|---------|
+| MAC Address      | macAddress      | The MAC address of the network host.                                   | Yes      |         |
```

Comment:
If I understand it correctly, mDNS will take precedence over MAC address in case of having randomly generated MAC addresses? In this case, is there any reason why MAC address is required, if you would just have to apply a dummy value? So perhaps the logic could be to technically have them both as optional, but validate that at least one of them is set. WDYT?

Reply:
Yes, this was my initial plan but the MAC remains the corner stone of the api requests, so impacts on code would have been very extensive (e.g. discovery remains MAC driven). In this approach mDNS is a complement to help rediscover MAC.